### ISSUE TITLE
An Issue with the "Bard and Dancer Spirits" Skill in Soul Linker

### ISSUE THREAD

**Issue by changelets** [HerculesWS/Hercules#1953]
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Issue Prelude

[//]: # (Please complete these mandatory steps and check the following boxes by putting an `x` inside the brackets _before_ filing your issue)

-  [x] I have not modified the source prior to reproducing this issue.
-  [x] I am using the latest version of Hercules.
-  [x] I am aware that this report will be closed or deleted if it becomes obvious that I am stating the false.

### Description

Hi there,

Basically, the Bard and Dancer Spirits skill (A soul linker's skill) is supposed to increase the walking speed for the targeted bard or dancer while singing or dancing, however, that doesn't happen.

Thank you,

**Comment by EyesOfAHawk** [comment]
i thought it makes your speed normal so it doesnt slow you down
